.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ dev ]
+  pull_request:
+    branches: [ dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Build with Maven
```

Comment:
Impressive, I was adding this Automated action to run a Build Self-Check any time one of Commits changes to the Branch and sure it enough it proved to be useful out of the gate!

I renamed the Repo and thought I made all the changes required to make it clean based on the new name but it appears at least one area [core] has a POM file with references to the previous Repo name. 

This is a very good test of using GitHub Actions to automatically check our work. 

I will fix the problem on Monday.